dashboard/app/models/school.rb

```diff
@@ -426,7 +444,7 @@ def self.merge_from_csv(filename, options = CSV_IMPORT_OPTIONS, write_updates =
       if new_schools.any?
         summary_message <<
           "Schools#{future_tense_dry_run} added:\n"\
-          "#{new_schools.map {|school| school[:name] + ' ' + school[:id].to_s}.join("\n")}\n"
+          "#{new_schools.map {|school| (school[:name] || '') + ' ' + school[:id].to_s}.join("\n")}\n"
```

Comment:
if schools are required to have a name, when would we hit the || condition here?

Reply:
I hit it when dry running this import without `write_inserts=false` specified -- I think you're right, makes sense that you should get an error here if you're trying to insert rows and you don't have name in those rows. I'll remove.